reference/constraints/Email.rst

```diff
@@ -91,14 +91,34 @@ Basic Usage
 Options
 -------
 
+mode
+~~~~
+
+**type**: ``string`` **default**: ``loose``
+
+This option is optional and pattern the email address is validated against.
```

Comment:
This option is optional and defines the pattern the e-mail address is validated against.